authors/apps/articles/views.py

```diff
@@ -142,3 +143,122 @@ def put(self, request, slug, *args, **kwargs):
                     serializer.errors,
                     status=400
                 )
+
+
+def get_article(slug):
+    """
+        Returns specific article using slug
+    """
+    article = Article.objects.all().filter(slug=slug).first()
+    if article is None:
+        raise exceptions.NotFound({
+            "message": error_msgs['not_found']
+        }, status.HTTP_404_NOT_FOUND)
+    return article
+
+
+class ShareViaEmail(generics.CreateAPIView):
+    """
+        Share Articles via email
+    """
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, slug):
```

Comment:
Good work but it would be better to also include a description instead of just the endpoint. Do it for the rest of the methods

Reply:
Okay, thank you. I have made changes as requested